app/views/home/index.html.slim

```diff
@@ -13,6 +13,8 @@
           }
         }
 
+  link rel="search" type="application/opensearchdescription+xml" title="Ruby API" href="/opensearch.xml"
```

Comment:
This looks good! My only suggestion is that it would be good to move this into `views/layouts/application.html.slim` so the browser can pickup the OpenSearch configuration for people who are landing on ruby doc pages from Search engines.

Reply:
I've moved it into there :)